source/includes/platform-payments.md.erb

```diff
@@ -199,28 +218,28 @@ In sandbox use this <a href="#payments-manager-put-companies-id-enable">endpoint
 | ukCompany.number  | string | The Company Number registered with Companies House.|
 | ukCompany.name    | string | Company Name.                                      |
 
-## `PUT /companies/{id}/enable 🚧`
+## `PUT /test/companies/{id}/enable 🚧`
```

Comment:
I've changed this but I've added and documented this endpoint couple days ago.
Nobody is probably using it, I'll ask Simon did he announced it or check the logs for usage.